opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchGeoPointType.java

```diff
@@ -22,6 +26,9 @@ public class OpenSearchGeoPointType extends OpenSearchDataType {
   private OpenSearchGeoPointType() {
     super(MappingType.GeoPoint);
     exprCoreType = UNKNOWN;
+    this.properties = new HashMap<>();
```

Comment:
Just FYI: `HashMap` can randomly reorder entries.
You are not affected unless you serialize it into string or iterate by index (not by key)

Reply:
Thanks changed in [920c17c](https://github.com/opensearch-project/sql/pull/1910/commits/920c17c8cf76709a1d4f1b26505ed2c1fd9b2db7)